vizro-core/examples/scratch_dev/app.py

```diff
@@ -3,59 +3,432 @@
+                va.set_control(control=f"{pre}filter_3", value=None),
+                va.set_control(control=f"{pre}filter_4", value=None),
+            ],
+        ),
+        vm.Button(
+            text="Set []",
+            actions=[
+                va.set_control(control=f"{pre}filter_1", value=[]),
+                va.set_control(control=f"{pre}filter_2", value=[]),
+                va.set_control(control=f"{pre}filter_3", value=[]),
+                va.set_control(control=f"{pre}filter_4", value=[]),
+            ],
+        ),
+        vm.Card(
+            text="Set 'versicolor'",
+            actions=_create_set_control_actions(prefix=pre, value="versicolor"),
+        ),
+        vm.Card(
+            text="""Set [['setosa', 'versicolor']]""",
+            actions=_create_set_control_actions(prefix=pre, value=["setosa", "versicolor"]),
+        ),
+        vm.AgGrid(id=f"{pre}table", figure=dash_ag_grid(px.data.iris())),
+    ],
+    controls=_create_filters(prefix=pre),
+)
+
+pre = "p5_"
+pre_target = "p6_"
+page_5 = vm.Page(
+    title="Drill-through source page",
+    layout=vm.Grid(grid=[[0, 1]]),
     components=[
         vm.Container(
-            title="Button Styles",
-            layout=vm.Flex(direction="row"),
+            title="Set multi-select",
+            layout=vm.Flex(),
             components=[
-                vm.Button(text="Primary", variant="filled"),
-                vm.Button(text="Primary", icon="Download", variant="filled"),
-                vm.Button(text="", icon="Download", variant="filled"),
-                vm.Button(text="Secondary", variant="outlined"),
-                vm.Button(text="Secondary", icon="Download", variant="outlined"),
-                vm.Button(text="", icon="Download", variant="outlined"),
-                vm.Button(text="Tertiary", variant="plain"),
-                vm.Button(text="Tertiary", icon="Download", variant="plain"),
-                vm.Button(text="", icon="Download", variant="plain"),
+                vm.Graph(
+                    figure=px.bar(
+                        px.data.iris(), x="sepal_width", y="sepal_length", color="species", custom_data=["species"]
+                    ),
+                    actions=va.set_control(control=f"{pre_target}filter_1", value="species"),
+                ),
+                vm.Button(
+                    text="Set None",
+                    actions=va.set_control(control=f"{pre_target}filter_1", value=None),
+                ),
+                vm.Button(
+                    text="Set []",
+                    actions=va.set_control(control=f"{pre_target}filter_1", value=[]),
+                ),
+                vm.Card(
+                    text="Set 'versicolor'",
+                    actions=va.set_control(control=f"{pre_target}filter_1", value="versicolor"),
+                ),
+                vm.Card(
+                    text="""Set [['setosa', 'versicolor']]""",
```

Comment:
```suggestion
                    text="""Set \['setosa', 'versicolor'\]""",
```

I guess you were trying to escape the square brackets? The double square brackets doesn't work for that but I think this will.